README.md

```diff
@@ -83,6 +83,16 @@ Deploy the site by running:
 3. Visit [`0.0.0.0:8000/stylguide`](http://0.0.0.0:8000/styleguide) for the style guide
 4. Visit [`0.0.0.0:8000/a11y`](http://0.0.0.0:8000/a11y) for an accessibility report
 
+#### Tasks
+
+- `npm install` - Install local dependencies
+- `npm run build` - Builds the site and the styleguide
+- `npm test` - Builds the site, styleguide, and runs reports (a11y)
+- `npm start` - Watches and serves the site for local development
+- `npm run deploy` - Deploys the site to what is defined in the "deploy" configuration
```

Comment:
A little concerned about exposing this without a warning. We are promoting other flows for this project. In what context would the user want to deploy with this commend?

If this is run when the user is not on the master branch, won't it cause chaos?